TESTS/netsocket/dns/main.cpp

```diff
@@ -178,7 +178,9 @@ Case cases[] = {
 #endif
     Case("SYNCHRONOUS_DNS", SYNCHRONOUS_DNS),
     Case("SYNCHRONOUS_DNS_MULTIPLE", SYNCHRONOUS_DNS_MULTIPLE),
+#ifdef MBED_EXTENDED_TESTS
```

Comment:
where is `MBED_EXTENDED_TESTS` documented? I noticed few tests have this

Reply:
I don't think its documented anywhere currently. Temporarily moving the test to extended tests until we are able to solve the issues with it.

Reply:
It is documented in the test plan
https://github.com/ARMmbed/mbed-os/tree/master/TESTS/netsocket#building-test-binaries